### ISSUE TITLE
Can you press and release a key repeatedly?

### ISSUE THREAD

**Issue by toimine** [rvaiya/keyd#774]
I'm trying to replicate this script I had in AutoHotkey that did this to Z if you held LCtrl (I need it to avoid mashing the button in some games)
![image](https://github.com/rvaiya/keyd/assets/62563494/dc47a45c-a3c9-404b-b314-1129112cb0f7)
I've tried reading the manual, but apart from macros (?), I found nothing that would let me hold keys, or make them be held or released (maybe I overlooked something). I'm asking because games would ignore a lot of presses without actually holding the key for a while, as I've discovered with another key remapper.

**Comment by nsbgn** [comment]
No, I don't think that is currently possible. Would a global option to set the time between keydown/keyup of keypresses in a macro be sufficient for your use case?

(For future reference, this sounds like a problem similar to #732)

**Comment by toimine** [comment]
Ideally it would be nice to set separate lengths for key down and key up, but one time setting is okay too

**Comment by toimine** [comment]
> It's not clear how this differs from the behaviour of `macro2`. What is wrong with something like `leftcontrol = macro2(50, 100, z)`?

This line makes the key be pressed and released instantaneously, so the presses barely get registered by the game. What I'm looking for is setting up the whole cycle of the key getting pressed, held for some time, and released, can keyd do that? I haven't checked the docs in ages, but iirc I couldn't find anything like that.
